packages/cli/internal/pkg/cli/workflow/manager.go

```diff
@@ -118,6 +119,12 @@ type workflowOutputProps struct {
 	workflowRunLogOutputs map[string]interface{}
 }
 
+type manifestProps struct {
+	MainWorkFlowURL string   `json:"mainWorkFlowURL"`
+	InputFileURLs   []string `json:"inputFileURLs"`
+	EngineOptions   string   `json:"engineOptions"`
```

Comment:
nit: these struct member variable should follow camelCase.

Reply:
JSON struct members have to start with uppercase, as stated here:  ["Here, we have to begin struct field names with an uppercase English letter to make those fields exportable to other packages. If your struct contains a field that starts with a lowercase letter, the encoding/json package won’t include the particular field for the encoding process without throwing any error."](https://blog.logrocket.com/using-json-go-guide/)